compareECLFiles.cmake

```diff
@@ -12,6 +22,8 @@ set(BASE_RESULT_PATH ${PROJECT_BINARY_DIR}/tests/results)
 # Input:
 #   - casename: basename (no extension)
 #
+# Details:
+#   - This test class compares output from a simulation to reference files.
 macro (add_test_compareECLFiles casename filename simulator)
```

Comment:
i don't see the big gain, but okay, i will pass the prefix to the test name as a parameter to avoid some c&p.